services/121-service/src/payments/reconciliation/safaricom-reconciliation/safaricom-reconciliation.controller.ts

```diff
@@ -31,6 +33,9 @@ export class SafaricomReconciliationController {
     status: HttpStatus.CREATED,
     description: 'Notified transfer status',
   })
+  @UsePipes(
+    new ValidationPipe({ whitelist: false, forbidNonWhitelisted: false }),
+  ) // Registrations have dynamic attributes, so we cannot use whitelist
```

Comment:
The comment does not fit here. (and in more places I think)

I'll make a task to also do this for onafriq.